src/Highlighter.php

```diff
@@ -41,6 +42,7 @@ public function __construct(
             ->addLanguage(new CssLanguage())
             ->addLanguage(new DiffLanguage())
             ->addLanguage(new DocCommentLanguage())
+            ->addLanguage(new EllisonLanguage())
```

Comment:
I'm not sure whether it should be added by default if the required package is opt-in. However, let's keep it like this for now, and we'll see whether it gives any problems :)